### PR TITLE
Make PHP 8 build green

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: Tests/RegistryTest.php [hunk 1]
```diff
@@ -140,7 +138,7 @@ public function testReset() : void
         $noProxyManager->expects($this->once())
             ->method('clear');
 
-        $proxyManager = $this->getMockBuilder([LazyLoadingInterface::class, EntityManagerInterface::class])->getMock();
+        $proxyManager = $this->createMock(LazyLoadingEntityManagerInterface::class);
```
Comment (SenseException) [inline Tests/RegistryTest.php:141]: Why did you create `LazyLoadingEntityManagerInterface`? Isn't the previous multi-interface mock creation not working anymore in PHP 8?

Comment (greg0ire) [reply Tests/RegistryTest.php:141]: It does not have to do with PHP 8, but with PHPUnit 9, which no longer supports it because it's a code smell IIRC.

Comment (greg0ire) [reply Tests/RegistryTest.php:141]: See https://github.com/sebastianbergmann/phpunit/issues/3955

Comment (SenseException) [reply Tests/RegistryTest.php:141]: Guess I have to remove another one of my old code snippets :smile:

**Review by SenseException — APPROVED** [review]

**Pull request merged**